app/celery/letters_pdf_tasks.py

```diff
@@ -213,8 +215,14 @@ def process_virus_scan_passed(self, filename):
         _move_invalid_letter_and_update_status(notification, filename, scan_pdf_object)
         return
 
-    new_pdf = _sanitise_precompiled_pdf(self, notification, old_pdf)
-
+    sanitise_response = _sanitise_precompiled_pdf(self, notification, old_pdf)
+    if not sanitise_response:
```

Comment:
sanitise_response is never empty so this wouldn't set `new_pdf`. Do we have a test for this?

Reply:
It's empty when validation has failed, then `_sanitise_precompiled_pdf()` returns None.